src/aap_eda/services/ruleset/activation_kubernetes.py

```diff
@@ -415,19 +415,21 @@ def set_activation_status(
         self, instance: models.ActivationInstance, status: ActivationStatus
     ) -> None:
         try:
+            now = timezone.now()
             instance.status = status
-            instance.updated_at = timezone.now()
+            instance.updated_at = now
             instance.save(update_fields=["status", "updated_at"])
 
-            update_fields = ["status", "modified_at"]
+            update_fields = ["status", "modified_at", "status_updated_at"]
```

Comment:
Wouldn't it better to override the "save" method for the activation model to update "status_updated_at" every time "status" is updated?

Reply:
The value can be set by an external value https://github.com/ansible/eda-server/blob/main/src/aap_eda/wsapi/consumers.py#L156

Reply:
gotcha, fair point. That logic is a different discussion :)